linera-service/src/project.rs

```diff
@@ -298,4 +359,53 @@ impl Project {
         file.write_all(content.as_bytes())?;
         Ok(())
     }
+
+    /// Adds ['linera-sdk'] dependencies in `debug` mode.
+    ///
+    /// Uses the directory of `linera-service` at compile time to figure out
+    /// where `linera-sdk` is.
+    #[cfg(debug_assertions)]
```

Comment:
Maybe this should be a separate feature flag?
I think it's surprising if the behavior changes in such a way based on whether optimizations are enabled.

Reply:
As discussed I don't have a better idea either: The added complexity of another feature flag might be worse.